connect/src/spirc.rs

```diff
@@ -611,6 +626,12 @@ impl SpircTask {
                 }
             }
             SpircCommand::Shutdown => {
+                self.player.emit_session_connect_event(
+                    self.session.connection_id(),
+                    self.session.username(),
+                    false,
```

Comment:
Could we also have an `emit_session_disconnect_event` instead of this `bool`? First a personal style preference, second it may help to implement logout commands in the future.

Reply:
Sure, no problem.

Reply:
Done